#ifndef _API_HANDLERS_HPP
#define _API_HANDLERS_HPP

#include <api/catalog/handlers.hpp>
#include <api/event/handlers.hpp>
#include <api/geo/handlers.hpp>
#include <api/kvdb/handlers.hpp>
#include <api/policy/handlers.hpp>
#include <api/router/handlers.hpp>
#include <api/tester/handlers.hpp>

#endif // _API_HANDLERS_HPP
